docs/manual/annotating-libraries.tex

```diff
@@ -992,9 +992,9 @@
 from ajava files into Java files.  This can be used to
 insert the annotations inferred by whole program inference into the source code
 of a program, similarly to the
-\href{https://checkerframework.org/annotation-file-utilities/#insert-annotations-to-source}{\code{insert-annotations-to-source}
+\href{https://eisop.github.io/afu/annotation-file-utilities.html/#insert-annotations-to-source}{\code{insert-annotations-to-source}
```

Comment:
```suggestion
\href{https://eisop.github.io/afu/#insert-annotations-to-source}{\code{insert-annotations-to-source}
```

Reply:
This works for the `../cf/manual/#section` links, but doesn't seem to work here. Wouldn't this be the same?